examples/step-19/doc/intro.dox

```diff
@@ -0,0 +1,459 @@
+(the "anode"). The anode is typically ring-shaped so that the majority of
+electrons can fly through the hole in the form of an electron beam. In the olden
+times, they might then have illuminated the screen of a TV built from a
+[cathode ray tube](https://en.wikipedia.org/wiki/Cathode-ray_tube).
+Today, instead, electron beams are useful in
+[X-ray machines](https://en.wikipedia.org/wiki/X-ray_tube),
+[electron beam lithography](https://en.wikipedia.org/wiki/Electron-beam_lithography),
+[electron beam welding](https://en.wikipedia.org/wiki/Electron-beam_welding), and
+a number of other areas.
+
+The equations we will then consider are as follows: First, we need to describe
+the electric field. This is most easily accomplished by noting that the electric
+potential $V$ satisfied the equation
+@f[
+  -\epsilon_0 \Delta V = \rho
+@f]
+where $\epsilon_0$ is the dielectric constant of vacuum, and $\rho$ is the charge
+density. This is augmented by boundary conditions that we will choose as follows:
+@f{align*}{
+  V &= -V_0 && \text{on}\; \Gamma_\text{cathode}\subset\partial\Omega \\
+  V &= +V_0 && \text{on}\; \Gamma_\text{anode}\subset\partial\Omega \\
+  \epsilon\frac{\partial V}{\partial n} &= 0
+   && \text{on}\; \partial\Omega\setminus\Gamma_\text{cathode}\setminus\Gamma_\text{anode}.
+@f}
+In other words, we prescribe voltages $+V_0$ and $-V_0$ at the two electrodes
+and isolating (Neumann) boundary conditions elsewhere. Since the dynamics of the
+particles are purely due to the electric field $\mathbf E=\nabla V$, we could
+as well have prescribed $2V_0$ and $0$ at the two electrodes -- all that matters
+is the voltage difference at the two electrodes.
+
+Given this electric potential $V$ and the electric field $\mathbf E=\nabla V$,
+we can describe the trajectory of the $i$th particle using the differential
+equation
+@f[
+  m {\ddot {\mathbf x}}_i = e\mathbf E,
+@f]
+where $m,e$ are the mass and electric charge of each particle. In practice, it
+is convenient to write this as a system of first-order differential equations
+in the position $\mathbf x$ and velocity $\mathbf v$:
+@f{align*}{
+  {\dot {\mathbf v}}_i &= \frac{e\mathbf E}{m}, \\
+  {\dot {\mathbf x}}_i &= {\mathbf v}_i.
+@f}
+The deal.II class we will use to deal with particles, Particles::ParticleHandler,
+stores particles in a way so that the position $\mathbf x_i$ is part of the
+Particles::ParticleHandler data structures. (It stores particles sorted
+by cell they are in, and consequently needs to know where each particle is.)
+The velocity $\mathbf v_i$, on the other hand, is of no concern to
+Particles::ParticleHandler and consequently we will store it as a
+"property" of each particle that we will update in each time step. Properties
+can also be used to store any other quantity we might care about each particle:
+its charge, or if they were larger than just an electron, its color, mass,
+attitude in space, chemical composition, etc.
```

Comment:
```suggestion
altitude in space, chemical composition, etc.
```

Reply:
No, I actually did mean "attitude: A vehicle's orientation about its center of gravity, known as its attitude in flight dynamics" :-)